### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by AlexandraBozhan**

### CODE REVIEW

File: src/getHumanAge.test.js [hunk 1]
```diff
@@ -8,6 +8,22 @@ describe(`Function 'getHumanAge':`, () => {
...
+    expect(getHumanAge(15, 24)).toEqual([1, 2]);
+  });
+
+  it(`should return an array [3, 6] if catAge = 28, dogAge = 44`, () => {
+    expect(getHumanAge(28, 44)).toEqual([3, 6]);
+  });
 
+  it(`should return an array [0, 0] if catAge = '', dogAge = ''`, () => {
+    expect(getHumanAge('', '')).toEqual([0, 0]);
   });
```
Comment (brainwashers) [inline src/getHumanAge.test.js:28]: how about to check boundary values?
